i18n/en/docusaurus-plugin-content-docs/current/14python-implementation/python-implementation.md

```diff
@@ -10,19 +10,19 @@ https://github.com/Chia-Network/chia-blockchain
 
 The codebase is split into several subfolders which categorize code on the different components of the system. Inside the source root (chia folder).
 
-* **cmds**: Contains the command line interface program, which is a python wrapper around the chia RPC api. When a command is run, a connection is established directly to the Chia services running locally.
+* **cmds**: Contains the command line interface program, wihch is a python wrapper around the chia RPC api. When a command is run, a connection is established directly to the Chia services running locally.
```

Comment:
This file is all reversions from the fixes.

Reply:
whoops, i meant to just do a manual pull of the chinese translations. I also need to update crowdin with the new fixes